Paws4Me/Paws4Me/Display All Adoptable Pets/Repository/Repositable.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Repositable.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/03/24.
+//
+
+import Foundation
+
```

Comment:
I think for now lets keep this in the same class as the repository 😄

Reply:
Will do so 👍🏻